pkg/collector/osmlogs_collector.go

```diff
@@ -140,9 +158,17 @@ func collectControllerLogs(collector *OsmLogsCollector, rootPath, meshName strin
 
 // Helper function to get all resoures of given type in the cluster
 func getResourceList(resource, label, outputFormat, separator string) ([]string, error) {
-	resourceList, err := utils.RunCommandOnContainer("kubectl", "get", resource, "--all-namespaces", "--selector", label, outputFormat)
+	outputStreams, err := utils.RunCommandOnContainerWithOutputStreams("kubectl", "get", resource, "--all-namespaces", "--selector", label, outputFormat)
 	if err != nil {
 		return nil, err
 	}
+
+	resourceList := outputStreams.Stdout
+	// If the resource is not found within the cluster, then log a message and do not return any resources.
+	if len(resourceList) == 0 {
+		log.Printf("Collector osmlogs: No '%s' resource with the label '%s' found in the cluster.\n", resource, label)
+		return nil, err
```

Comment:
This err will be the err from 
`outputStreams, err := utils.RunCommandOnContainerWithOutputStreams("kubectl", "get", resource, "--all-namespaces", "--selector", label, outputFormat)`
We should make a new error something like	
`err :=  fmt.Errorf("No %s resource found in cluster", resource)`